source/MaterialXGenMdl/MdlShaderGenerator.cpp

```diff
@@ -367,6 +357,137 @@ string MdlShaderGenerator::getUpstreamResult(const ShaderInput* input, GenContex
     return variable;
 }
 
+
+namespace
+{
+    // [TODO]
+    // Here we assume the MSB of the port flags is unused.
```

Comment:
Maybe better to expand MSB vs using an acronym. I don't recall what this means.

Reply:
MSB = Most Significant Bit, but I'll update the comment.